README.md

```diff
@@ -24,4 +24,10 @@ occ config:app:set dashboard layout --value=welcome,recommendations,spreed,mail,
 
 ### Screenshot
 
-![Welcome widget example](img/screenshot1.jpg)
\ No newline at end of file
+![Welcome widget example](img/screenshot1.jpg)
+
+### General Usage Guide
+
+1. The markdown file should only be created from the Text app for proper support of links and images
+2. Manually added links should ideally be absolute links with file IDs for files/folders
```

Comment:
Do you mean `/f/` link? I mean, "absolute link" is ambiguous (at least for me :grin: ). Could you be more specific?

Reply:
:saluting_face: